cli_tools/gce_windows_upgrade/main.go

```diff
@@ -0,0 +1,100 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// gce_windows_upgrade is a tool for upgrading GCE Windows instances.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/logging/service"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_windows_upgrade/upgrader"
+)
+
+var (
+	clientID               = flag.String(upgrader.ClientIDFlagKey, "", "Identifies the upgrade client. Set to `gcloud`, `api` or `pantheon`.")
+	project                = flag.String("project", "", "Project containing the instance to upgrade. Override by using a different project in the -instance parameter.")
+	zone                   = flag.String("zone", "", "Project containing the instance to upgrade. Override by using a different zone in the -instance parameter.")
+	instance               = flag.String("instance", "", "Instance to upgrade. Can be either the instance name or the project path to the instance, format: 'projects/<project>/zones/<zone>/instances/<instance>'.")
```

Comment:
Proposed new text:

"Instance to upgrade. Can be either the instance name or the project path to the instance, format: 'projects/<project>/zones/<zone>/instances/<instance>'. Use this parameter to override the project and zone parameters.

Reply:
Discussed one more with Zoran, now the proposal is:
Instance to upgrade. Can be either the instance name or the full path to the instance in the following format: 'projects/<project>/zones/<zone>/instances/'. If the full path is specified, flags -project and -zone will be ignored.